src/vanilla/harmonium.js

```diff
@@ -0,0 +1,288 @@
+/**
+ * Closes the given flash component
+ * @param {HTMLElement} flashComponentElement The Flash component
+ * @returns {void}
+ */
+export function closeFlash(flashComponentElement) {
+  if (flashComponentElement) {
+    flashComponentElement.parentNode.removeChild(flashComponentElement)
+  }
+}
+
+/**
+ * Initializes all flash components on page.
+ * @returns {void}
+ */
+export function initializeFlashComponents() {
+  const components = document.querySelectorAll('.rev-Flash')
```

Comment:
wonder if we should opt for `data` attrs instead of css class names?

Reply:
It could be a future consideration. But the react components and css use class names so following that path

Reply:
Well the react components use class names only to reference the css, not to make handle the js interactions, but yea I feel you. No need to address atm